.github/workflows/pr.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: bcgov/openshift-launchpad-deployment@master
         with:
           MODE: client
-          BRANCH: ${{ github.event.pull_request.head.ref }}
+          BRANCH: "${{ github.event.pull_request.head.ref }}"
```

Comment:
Not really needed as it's not bash and going to resolve to a string anyway.

Reply:
Removed it.